CHANGELOG.md

```diff
@@ -5,7 +5,10 @@
 ### Enhancements
 
 - Cache parsed DSN ([#2365](https://github.com/getsentry/sentry-dart/pull/2365))
-
+- Handle backpressure earlier in pipeline ([#2371](https://github.com/getsentry/sentry-dart/pull/2371))
+  - Drops max un-awaited parallel tasks earlier, so event processors & callbacks are not executed for them. 
```

Comment:
what happens though if you `await Sentry.captureException` in a tight loop 100 times. Will those still be dropped? Since you mention here it only works for un-awaited parallel tasks

Reply:
No, they will not be dropped. Since we are always awaiting the next task, this should be fine from a back-pressure perspective.